pkg/controllers/provisioningv2/rke2/common.go

```diff
@@ -73,8 +75,18 @@ const (
 var (
 	ErrNoMachineOwnerRef = errors.New("no machine owner ref")
 	labelAnnotationMatch = regexp.MustCompile(`^((rke\.cattle\.io)|((?:machine\.)?cluster\.x-k8s\.io))/`)
+	windowsDrivers       = map[string]struct{}{
+		"vmwarevsphere": {},
+	}
 )
 
+// WindowsCheck return a bool based on if the driver is marked as
+// supporting Windows
+func WindowsCheck(driver string) bool {
+	_, ok := windowsDrivers[driver]
+	return ok
+}
+
```

Comment:
It looks like this is no longer used and can be removed.

Reply:
`WindowsCheck` is still used in dynamicschema https://github.com/rosskirkpat/rancher/blob/3ce8e4f7485c3df5bea5e9d1f103be6bed6617cf/pkg/controllers/provisioningv2/rke2/dynamicschema/dynamicschema.go#L59